app/tests/test_generic_api.py

```diff
@@ -491,15 +491,15 @@ class ApiCORSHeaderTestCase(StacBaseTestCase):
 
     @classmethod
     @mock_s3_asset_file
-    def setUpTestData(cls):
```

Comment:
Why changing it from `setUpTestData` to `setUp` ?

Reply:
It has to do with the "@mock_s3_asset_file" decoration, if I remember correctly. If I don't change this, the following error appears: `ValueError: I/O operation on closed file.` I couldn't totally understand the root of the problem, but if you think it's important, I can try to dig deeper into that.